### PR TITLE
feat(osd): Enable hitting multiple OSD endpoints

### PR DESCRIPTION
**Pull request by bradbeam**
This enables the ability to specify additional <talos> endpoints to connect to
to pull back data.

Signed-off-by: Brad Beam <brad.beam@talos-systems.com>

### CODE REVIEW

**Comment by bradbeam** [comment]
s/target/endpoint

s/ralph/target

**Comment by bradbeam** [comment]
Ex:
```
[21:28:48] $ ./build/osctl-darwin-amd64 --target 10.5.0.2,10.5.0.3,10.5.0.4,10.5.0.5 containers
HOST       NAMESPACE   ID         IMAGE            PID   STATUS
10.5.0.5   system      networkd   talos/networkd   73    RUNNING
10.5.0.5   system      osd        talos/osd        216   RUNNING
HOST       NAMESPACE   ID         IMAGE            PID   STATUS
10.5.0.4   system      networkd   talos/networkd   93    RUNNING
10.5.0.4   system      osd        talos/osd        160   RUNNING
10.5.0.4   system      trustd     talos/trustd     130   RUNNING
HOST       NAMESPACE   ID         IMAGE            PID   STATUS
10.5.0.3   system      networkd   talos/networkd   103   RUNNING
10.5.0.3   system      osd        talos/osd        161   RUNNING
10.5.0.3   system      trustd     talos/trustd     96    RUNNING
HOST       NAMESPACE   ID         IMAGE            PID   STATUS
10.5.0.2   system      networkd   talos/networkd   80    RUNNING
10.5.0.2   system      osd        talos/osd        127   RUNNING
10.5.0.2   system      trustd     talos/trustd     129   RUNNING
```

File: cmd/osctl/cmd/containers.go [hunk 1]
```diff
@@ -55,25 +60,28 @@ var containersCmd = &cobra.Command{
...
+		resp := rep
+		sort.Slice(resp.Containers,
+			func(i, j int) bool {
+				return strings.Compare(resp.Containers[i].Id, resp.Containers[j].Id) < 0
+			})
 
-	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
-	fmt.Fprintln(w, "NAMESPACE\tID\tIMAGE\tPID\tSTATUS")
+		w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
+		fmt.Fprintln(w, "HOST\tNAMESPACE\tID\tIMAGE\tPID\tSTATUS")
```
Comment (andrewrynhard) [inline cmd/osctl/cmd/containers.go]: Let's use the Kubernetes nomenclature and rename the `HOST` column to `NODE`.

File: cmd/osctl/cmd/processes.go [hunk 2]
```diff
@@ -178,33 +188,36 @@ func processesOutput(ctx context.Context, c *client.Client) (output string, err
...
+			by(cpu).sort(procs)
 		default:
-			args = p.Args
+			by(rss).sort(procs)
 		}
 
-		s = append(s,
-			fmt.Sprintf("%6d | %1s | %4d | %8.2f | %7s | %7s | %s",
-				p.Pid, p.State, p.Threads, p.CpuTime, bytefmt.ByteSize(p.VirtualMemory), bytefmt.ByteSize(p.ResidentMemory), args))
+		s = append(s, "HOST | PID | STATE | THREADS | CPU-TIME | VIRTMEM | RESMEM | COMMAND")
```
Comment (andrewrynhard) [inline cmd/osctl/cmd/processes.go]: Let's use the Kubernetes nomenclature and rename the HOST column to NODE.

**Pull request merged**